packages/terra-docs/src/Notice.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './Notice.module.scss';
+
+const cx = classNames.bind(styles);
+
+const getTitle = (variant) => {
+  if (variant === 'best-practice') {
+    return 'Best Practice';
+  }
+
+  if (variant === 'caution') {
+    return 'Caution';
+  }
+
+  if (variant === 'deprecation') {
+    return 'Deprecated';
+  }
+
+  if (variant === 'maintenance') {
+    return 'In Maintenance';
+  }
+
+  if (variant === 'note') {
+    return 'Note';
+  }
```

Comment:
should we provide notice for MVB's or that will be taken care as part in one of the above notice types.

Reply:
I'm not sure how you'd use a notice to document a MVB. Say you've got a component that you version from 1.x to 2.x. You make the changes to the doc and publish the updated documentation for 2.x. Do you place a notice at the top of the 2.x component doc Stating that It's been upgraded to v2? How long do you keep the notice in documentation? I have zero faith in us going back and removing that notice on any type of schedule.

I agree that we should find a way to raise awareness of major version changes, but I don't think this notice component can help with that problem.

Reply:
yeah i agree. we should probably think of notifications in our dev-site for components to notify the consumers on MVB's.